### PR TITLE
initial stab at requiring adminness

### PR DESCRIPTION
**Pull request by anotherjesse**
haven't hit everything but I'm having to catch:

```
client_exceptions.ClientException
```

instead of:

```
client_exceptions.Unauthorized
```

Since right now these seem to throw 500s

Also not sure what to do about the CLI version of these tests since it throws:

```
ERROR: test_admin_requires_adminness (test_cli.CliMasterTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/jesse/rcb/keystonelight/tests/test_keystoneclient.py", line 382, in test_admin_requires_adminness
    self.assertRaises(exception, two.users.get, self.user_two['id'])
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/unittest/case.py", line 456, in assertRaises
    callableObj(*args, **kwargs)
TypeError: __call__() takes exactly 1 argument (2 given)
```

### CODE REVIEW

File: tests/test_keystoneclient.py [hunk 1]
```diff
@@ -369,3 +369,41 @@ def test_service_list(self):
         services = client.services.list()
         # TODO(devcamcar): This assert should be more specific.
         self.assertTrue(len(services) > 0)
+
+    def test_admin_requires_adminness(self):
+        from keystoneclient import exceptions as client_exceptions
+        # FIXME(termie): this should be Unauthorized
```
Comment (termie) [inline tests/test_keystoneclient.py]: you can do FIXME or TODO with your own name, not somebody else's ;)

File: tests/test_keystoneclient.py [hunk 2]
```diff
@@ -369,3 +369,41 @@ def test_service_list(self):
...
+        exception = client_exceptions.ClientException
+
+        two = self.get_client(self.user_two)  # non-admin user
+
+        # USER CRUD
+        self.assertRaises(exception, two.users.list)
+        self.assertRaises(exception, two.users.get, self.user_two['id'])
+        self.assertRaises(exception, two.users.create, name='oops',
```
Comment (termie) [inline tests/test_keystoneclient.py]: this look slike it is has gotten to the one argument per line length

File: tests/test_keystoneclient.py [hunk 3]
```diff
@@ -369,3 +369,41 @@ def test_service_list(self):
...
+        # NOTE(ja): tenants.list is different since /tenants fulfills the
+        #           two different tasks: return list of all tenants & return
+        #           list of tenants the current user is a member of...
+        #           which means if you are admin getting the list
+        #           of tenants for admin user is annoying?
+        tenants = two.tenants.list()
+        self.assertTrue(len(tenants) == 1)
+        self.assertTrue(tenants[0].id == self.tenant_baz['id'])
+        self.assertRaises(exception, two.tenants.get, self.tenant_bar['id'])
+        self.assertRaises(exception, two.tenants.create,
```
Comment (termie) [inline tests/test_keystoneclient.py]: one arg per line

**Comment by termie** [comment]
to fix the cli test just pass all the args as keyword args instead of positionals (this is sort of annoying since keystoneclient isn't particularly consistent with its arg naming)

File: tests/test_keystoneclient.py (hunk 1 above)
Comment (anotherjesse) [inline tests/test_keystoneclient.py]: but I wanted you to fix it ;)  

done

File: tests/test_keystoneclient.py (hunk 2 above)
Comment (anotherjesse) [inline tests/test_keystoneclient.py]: fixed

File: tests/test_keystoneclient.py (hunk 3 above)
Comment (anotherjesse) [inline tests/test_keystoneclient.py]: fixed

**Comment by anotherjesse** [comment]
fixed cli tests - they pass - although keystoneclient is inconsistent on requiring foo vs foo_id :(

**Pull request merged**